generated/nidaqmx/_task_modules/channels/ai_channel.py

```diff
@@ -2619,7 +2618,7 @@ def ai_current_units(self):
         check_for_error(error_code)
 
     @property
-    def ai_custom_scale(self):
+    def ai_custom_scale_name(self):
```

Comment:
I think we need to figure out how/where the old generator renamed these properties and add it into grpc-device-scrapigen, so that we fix up both the property names and any references to them in docstrings.

Reply:
I agree. This needs to be fixed to `ai_custom_scale`. I think custom logic in the generator, similar to `EXCLUDED_ATTRIBUTES`, that has a list of things we manually rename for Python is fine.